src/pal/src/exception/signal.cpp

```diff
@@ -927,6 +930,19 @@ void handle_signal(int signal_id, SIGFUNC sigfunc, struct sigaction *previousAct
     }
 #endif
 
+    if (skipIgnored)
+    {
+        if (-1 == sigaction(signal_id, NULL, previousAction))
+        {
+            ASSERT("handle_signal: sigaction() call failed with error code %d (%s)\n",
+            errno, strerror(errno));
```

Comment:
A nit - in this file we indent the second line of arguments in similar asserts by 4 chars. Could you please do it here as well? The change LGTM otherwise.